CvGameCoreDLL_Expansion2/Lua/CvLuaCity.cpp

```diff
@@ -1671,7 +1671,7 @@ int CvLuaCity::lGetGeneralProductionTurnsLeft(lua_State* L)
 //bool isFoodProduction();
 int CvLuaCity::lIsFoodProduction(lua_State* L)
 {
-	return BasicLuaMethod<bool>(L, &CvCity::isFoodProduction);
```

Comment:
you can avoid patching all these calls by putting the new template argument last instead of first

Reply:
I'll go ahead and change em all 👍